ksonnet-util/kausal.libsonnet

```diff
@@ -12,6 +12,12 @@ k {
       configMap+: {
         new(name)::
           super.new(name, {}),
+        withData(data)::
+          if (data == {}) then {}
+          else super.withData(data),
+        withDataMixin(data)::
+          if (data == {}) then {}
+          else super.withData(data),
```

Comment:
shouldn't this be `super.withDataMixin(data)` ?

Reply:
eeks, thanks for spotting @woodsaj .

Reply:
fix pushed